tox.ini

```
@@ -52,8 +70,12 @@ passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH
 deps =
 	pip-tools
 	coverage
+	-r requirements/tests.txt
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
-	pip-sync requirements/base.txt
 	pip install -e .
+	pip uninstall -y d3a-interface
+	pip install git+https://github.com/gridsingularity/d3a-interface.git@{env:BRANCH:master}
+;	flake8
+	coverage run -m py.test unit_tests/ --random-order {posargs:tests}
```

```suggestion
	coverage run -m pytest unit_tests/ --random-order {posargs:tests}
```
The `py.test` version is old and should be deprecated in the future.